fdbrpc/QueueModel.h

```diff
@@ -42,8 +64,22 @@ typedef double TimeEstimate;
 
 class QueueModel {
 public:
+	// Finishes the request sent to storage server with `id`.
+	//   - latency: the measured client-side latency of the request.
+	//   - penalty: the server side penalty sent along with the response from
```

Comment:
the indention looks funny. Did you run clang-format on the change?

Reply:
I just did it only in QueueModel.h. It also changes the code that I didn't change. I see that Aaron is doing a repo-wide clang-format change, so I'm not sure if I should add too much "unrelated" changes here.

Reply:
You can actually just clang-format your changed code. I used this one: https://github.com/barisione/clang-format-hooks